build/checks.mk

```diff
@@ -1,4 +1,4 @@
-ifneq (, $(shell /bin/bash -c "command -v git"))
+ifneq (, $(shell sh -c "command -v git"))
```

Comment:
Why not simply `$(shell command -v git)`?

Reply:
That was my first effort, it returns the following errors...

```bash
make: command: Command not found
make[1]: command: Command not found
make[2]: command: Command not found
...
```

In order for it to work correctly, it requires [modifying the SHELL variable in the Makefile](https://stackoverflow.com/questions/40182887/make-command-not-found). I was afraid of collateral damage, but I can avoid it by invoking the local shell.

Reply:
`$(shell command -v git 2> /dev/null)` ?

Reply:
That's weird. `command` should be supported everywhere. But if it doesn't work for you, it might not work for somebody else, so `sh -c command` sounds safer. :+1:

Reply:
Thanks @avtolstoy for sticking to your guns! I learned something new, thanks to the [link you shared with](https://stackoverflow.com/questions/17547625/how-to-use-shell-builtin-function-from-a-makefile) me about makefile shell execution.
I have updated the script according to your feedback, it looks much cleaner! Thanks for hanging with me. :smile: